Resources/Prototypes/Entities/Mobs/Player/jaunt_mobs.yml

```diff
@@ -0,0 +1,47 @@
+﻿- type: entity
+  name: jaunt
+  parent: [Incorporeal, BaseMob]
+  id: BaseMobJaunt
+  description: You shouldn't see this!
+  suffix: Ethereal
+  components:
+  - type: ContentEye
+    maxZoom: 1.44,1.44
+  - type: Eye
+    drawFov: false
+  - type: Input
+    context: "ghost"
+  - type: InputMover
+  - type: MovementSpeedModifier
+    baseSprintSpeed: 12
+    baseWalkSpeed: 8
+  - type: Physics
+    bodyType: KinematicController
+  - type: Visibility
+    layer: 2
+  - type: Spectral
+  - type: CanMoveInAir
+  - type: NoSlip
+  - type: Tag
+    tags:
+    - HideContextMenu
```

Comment:
This will also mean admins won't be able to interact with it via right clicking.
Since you can spawn the jaunt as a mob, I would maybe remove this.